src/Tests.IntegrationTests/Models/Institution.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Tests.IntegrationTests.Models
+{
+    public class Institution
```

Comment:
Consider changing this to `InstitutionResponse`

Reply:
I think the current name is correct as it represents the full `Institute` model. Keeping it names this way aligns better with how the `Participant` model is named. The request model has been renamed as suggested.